CBLClient/Apps/CBLTestServer-Java-WS/src/main/java/com/couchbase/javaws/TestServerContext.java

```diff
@@ -1,12 +1,36 @@
-package com.couchbase.javaws;
-
+package com.couchbase.mobiletestkit.javatestserver;
```

Comment:
java ties file structure with package name. if you make this change, you have to move the file from javws directory to the new directory you specified in package name, and other package reference in entire project will have to be updated too. I would suggest you not touch the package name.